### ISSUE TITLE
[dualtor] Incorrect DSCP field during decap flow

### ISSUE THREAD

**Issue by prsunny** [Azure/sonic-buildimage#7216]
#### Description

This is similar to #6653 but issue observed during decap. #6653 is resolved. 

#### Steps to reproduce the issue:
1. Configure tunnel 
```
 "SAI_TUNNEL_ATTR_DECAP_DSCP_MODE"
 "SAI_TUNNEL_DSCP_MODE_UNIFORM_MODEL"

```
2. Send encap traffic
3. Capture the decap packets

#### Describe the results you received:
Inner dscp value not modified and hence same value in decapsulated packet

#### Describe the results you expected:
Inner dscp value should be same as outer. 

#### Output of `show version`:

```
SONiC.20201231.beta.16
```
#### Output of `show techsupport`:

```
(paste your output here or download and attach the file here )
```

#### Additional information you deem important (e.g. issue happens only occasionally):
**Send packet:**
```

    00:00:aa:bb:cc:dd > 94:8e:d3:04:bf:d8, ethertype IPv4 (0x0800), length 74: (tos 0x0, ttl 64, id 1, offset 0, flags [none], proto IPIP (4), length 60)
    1.1.1.1 > 10.1.0.32: (tos 0x4, ttl 25, id 1, offset 0, flags [none], proto TCP (6), length 40)
    10.212.64.0.5000 > 192.168.0.10.8000: Flags [S], cksum 0x5194 (correct), seq 0, win 8192, length 0
	0x0000:  948e d304 bfd8 0000 aabb ccdd 0800 4500  ..............E.
	0x0010:  003c 0001 0000 4004 6e9b 0101 0101 0a01  .<....@.n.......
	0x0020:  0020 4504 0028 0001 0000 1906 9645 0ad4  ..E..(.......E..
	0x0030:  4000 c0a8 000a 1388 1f40 0000 0000 0000  @........@......
	0x0040:  0000 5002 2000 5194 0000                 ..P...Q...
```

**Received packet:**

```
    94:8e:d3:04:bf:d8 > 00:00:11:22:33:47, ethertype IPv4 (0x0800), length 60: (tos 0x4, ttl 24, id 1, offset 0, flags [none], proto TCP (6), length 40)
    10.212.64.0.5000 > 192.168.0.10.8000: Flags [S], cksum 0x5194 (correct), seq 0, win 8192, length 0
	0x0000:  0000 1122 3347 948e d304 bfd8 0800 4504  ..."3G........E.
	0x0010:  0028 0001 0000 1806 9745 0ad4 4000 c0a8  .(.......E..@...
	0x0020:  000a 1388 1f40 0000 0000 0000 0000 5002  .....@........P.
	0x0030:  2000 5194 0000 b392 fac1 0000            ..Q.........
```

**Comment by gechiang** [comment]
BRCM case CS00012182143:
[4.3][IPinIP][DECAP DSCP Uniform Mode] During Decap we observed DSCP Uniform mode is not working.  Expected outer DSCP was NOT propagated to the inner DSCP value.

Filed to track this issue.

**Comment by prsunny** [comment]
Closing as the fix is available in 4.3.3.9
